catalog/conformance-tests/result-carbontrail-001.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tested_solution": {
+    "solution_id": "carbontrail",
+    "version": "1.0.0"
+  },
+  "tested_by": {
+    "solution_id": "carboledger",
+    "version": "1.0.0",
```

Comment:
@shantanu1singh I believe versions should be 2.0.0 here as well both for the tested solution and the tested by.